src/types.ts

```diff
@@ -712,6 +712,7 @@ export enum FeatureName {
   UnifiedAlerting = 'ngalert',
   RBAC = 'synthetic-monitoring-rbac',
   AlertsPerCheck = 'sm-alerts-per-check',
+  SecretsManagement = 'synthetic-monitoring-secrets-management',
```

Comment:
Right now we have a mixture of names, but should we set on always prefixing feature flags with the `synthetic-monitoring-` prefix from now on? I know we have talked about this before @ckbedwell

Reply:
If it is a SM specific toggle, my vote is on `synthetic-monitoring-*`.
It makes it VERY clear to others.